src/app/core/permissions/permissions.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { DisableEntityOperationDirective } from "./disable-entity-operation.directive";
+import { DisabledWrapperComponent } from "./disabled-wrapper/disabled-wrapper.component";
+import { MatTooltipModule } from "@angular/material/tooltip";
+
+@NgModule({
+  declarations: [DisableEntityOperationDirective, DisabledWrapperComponent],
+  imports: [CommonModule, MatTooltipModule],
+  exports: [DisableEntityOperationDirective, DisabledWrapperComponent],
```

Comment:
should `DisabledWrapperComponent` rather stay internal and not be exported?

Reply:
True that. I guess it is not required